elixir/apps/web/lib/web/components/form_components.ex

```diff
@@ -137,6 +137,12 @@ defmodule Web.FormComponents do
     ~H"""
     <div phx-feedback-for={@name}>
       <.label for={@id}><%= @label %></.label>
+      <input
```

Comment:
`<select>` does not send the data if it's disabled, so we add a hidden input that will do the job for it.